### DISCUSSION TITLE
No stratum for payout

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1159] [Ideas]
We have set up a dedicated payout instance. The problem now is, that I still have to define a full pool including stratum proxy ports etc. 
This leads, at least for Ethereum to the problem, that I have to set up a (hidden) full node for payouts. 

If it is possible to have a dedicated payout process it should be possible to run this without any stratum proxy ports etc. and allow a (hidden) light wallet for payouts. Also nice would be a resupport of the old "coinbasePassword" in the config. With this, the payout can unlock geth and it hasn't to be run with --allow-insecure-unlock and a password file.

**Comment by oliverw** [comment]
Honestly I'm reluctant to put this much development effort into a coin that's going POS, at least for a community solution.
